packages/openactive-integration-tests/test/features/README.md

```diff
@@ -20,6 +20,7 @@ The tests for these features provide partial coverage but do not include all kno
 
 | Category | Feature | Specification | Description | Prerequisites per Opportunity Type |
 |----------|---------|---------------|-------------|-------------------|
+| core | Amending the OrderQuote before B ([amending-order-quote](./core/amending-order-quote/README.md)) | [Required](https://www.openactive.io/open-booking-api/EditorsDraft/#amending-the-orderquote-before-b) | Allows the basket to be updated for a particular order | [TestOpportunityBookable](https://openactive.io/test-interface#TestOpportunityBookable) x8 |
```

Comment:
Note that it's `x8` due to the `numOpportunitiesUsedPerCriteria: 2`

Reply:
👍